src/libsystemd-network/sd-radv.c

```diff
@@ -432,6 +432,24 @@ _public_ int sd_radv_set_ifindex(sd_radv *ra, int ifindex) {
         return 0;
 }
 
+int sd_radv_set_ifname(sd_radv *ra, const char *ifname) {
+        assert_return(ra, -EINVAL);
+        assert_return(ifname, -EINVAL);
+
+        if (!ifname_valid(ifname))
+                return -EINVAL;
```

Comment:
Maybe make this a different return code so it's possible to distinguish this important case?

Reply:
I have no good alternative... Any suggestion?

Reply:
EXDEV 18 Invalid cross-device link ?